google-cloud-bigtable/lib/google/cloud/bigtable/data_client.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require "google/cloud/bigtable"
+require "google/cloud/bigtable/v2"
+require "google/cloud/bigtable/table_data_operations"
+
+module Google
+  module Cloud
+    module Bigtable
+      class DataClient
+        # Bigtable client for data oprations.
+
+        attr_reader :options, :project_id, :instance_id
```

Comment:
These attributes need documentation. Also, the `options` attribute seems like it should be private.